Makefile

```diff
@@ -39,28 +49,40 @@ ${PROGNAME}: ${OBJ} ${PROGNAME}.c ${HEADERS}
 	${CC} ${FLAGS} -o ${PROGNAME} ${PROGNAME}.c ${LIBS}
 
 .PHONY: dist
-dist: ${MAIN_LIB} ${PROGNAME} 
+dist: 
+	@echo -e "\033[1m== Creating tar archive $(PROGNAME)-$(distversion).tar.gz ==\033[0;0m"
 	make clean
-	tar czf ../$(PROGNAME)-$(VERSION).tgz --transform='s,^${PROGNAME},$(PROGNAME)-$(VERSION),S' \
+	tar czf ../$(PROGNAME)-$(distversion).tar.gz \
 	--exclude=.git \
 	--exclude=.gitignore \
 	--exclude=README.md \
-	-C .. ${PROGNAME}
-	@mv ../$(PROGNAME)-$(VERSION).tgz ./
+	--transform='s,^\./,ebiso-0.1.1/,S' .
+	@mv ../$(PROGNAME)-$(distversion).tar.gz ./
 
 .PHONY: clean
 clean:
 	rm -f ${LIBDIR}/*.o ${LIBDIR}/*.a
 	rm -f ${PROGNAME}
-	rm -f $(PROGNAME)*.tgz
+	rm -f $(PROGNAME)*.tar.gz
 	rm -rf ${BASE_DEPDIR}
 
+.PHONY: rpm
+rpm: dist $(specfile)
+	@echo -e "\033[1m== Building RPM package $(PROGNAME)-$(distversion) ==\033[0;0m"
+	rpmbuild -v --clean  \
+		--define "_rpmfilename %%{NAME}-%%{VERSION}-%%{RELEASE}.%%{ARCH}.rpm" \
+		--define "debug_package %{nil}" \
+		--define "_rpmdir %(pwd)" \
+		-tb  $(PROGNAME)-$(distversion).tar.gz
+
 .PHONY: install
 install: ${PROGNAME}
-	install -m 0755 ${PROGNAME} ${INSTDIR}
+	@echo -e "\033[1m== Installing $(PROGNAME)-$(distversion) ==\033[0;0m"
+	[ ! -d $(DESTDIR)$(INSTDIR) ] && mkdir -m 755 -p $(DESTDIR)$(INSTDIR)
```

Comment:
@gdha For some strange reason I get error for following piece of code: 
```[ ! -d $(DESTDIR)$(INSTDIR) ] && mkdir -m 755 -p $(DESTDIR)$(INSTDIR)```

```
uefi:(/tmp/gdha/ebiso)(root)# make install
== Installing ebiso-0.1.1 ==
[ ! -d /usr/bin ] && mkdir -m 755 -p /usr/bin
make: *** [install] Error 1
```

When I use this  instead all seems to be OK:
```if [ ! -d $(DESTDIR)$(INSTDIR) ]; then mkdir -m 755 -p $(DESTDIR)$(INSTDIR); fi```
```
uefi:(/tmp/gdha/ebiso)(root)# make install
== Installing ebiso-0.1.1 ==
if [ ! -d /usr/bin ]; then mkdir -m 755 -p /usr/bin; fi
install -m 0755 ebiso /usr/bin
```
Any clue what can be wrong?

I'm using make-3.81-128.20 ...